install/kubernetes/quick-install.yaml

```diff
@@ -650,16 +642,3 @@ spec:
       restartPolicy: Always
       serviceAccount: cilium-operator
       serviceAccountName: cilium-operator
-
```

Comment:
Which version of `helm` did you use to generate this? Just wondering if we will keep toggling these lines at the end. I'm not sure which version generates these.

Reply:
ah, good point. I'm using v3.0.2. Maybe we should pin this somewhere? Where?

Reply:
OK, so long as you're on helm3 I don't expect this to cause more churn going forward. I expect that all Cilium developers will switch over shortly anyway.